packages/cfd/src/Components/cfd-personal-details-form.tsx

```diff
@@ -275,38 +266,31 @@ const CFDPersonalDetailsForm = ({
                 isValid,
             }: FormikProps<TFormValues>) => {
                 const citizenship_error = touched.citizen && errors.citizen;
-                const place_of_birth_error =
-                    is_in_personal_details_modal && touched.place_of_birth && errors.place_of_birth;
+                const place_of_birth_error = touched.place_of_birth && errors.place_of_birth;
                 const tax_residence_error = touched.tax_residence && errors.tax_residence;
                 const account_opening_reason_error = touched.account_opening_reason && errors.account_opening_reason;
-                const is_citizenship_disabled = !!(value.citizen && is_fully_authenticated);
+
+                const is_citizenship_disabled = !!value.citizen && !changeable_fields?.includes('citizen');
                 const is_place_of_birth_disabled =
-                    !!(value.place_of_birth && is_fully_authenticated) ||
-                    changeable_fields?.every(field => field !== 'place_of_birth');
-                const is_tax_residence_disabled = !!(value.tax_residence && is_fully_authenticated);
+                    !!value.place_of_birth && !changeable_fields?.includes('place_of_birth');
+                const is_tax_residence_disabled =
+                    !!value.tax_residence && !changeable_fields?.includes('tax_residence');
+                const is_account_opening_reason_disabled =
+                    !!value.account_opening_reason && !changeable_fields?.includes('account_opening_reason');
                 const handleItemSelection = (item: ResidenceList[0], _field: string) => {
                     const item_value = item.value ? item.text : '';
                     setFieldValue(_field, item_value, true);
                 };
 
                 return (
                     <AutoHeightWrapper default_height={200} height_offset={isDesktop() ? 148 : null}>
-                        {({
-                            height,
-                            setRef,
-                        }: {
-                            height: number;
-                            setRef: (instance: HTMLFormElement | null) => void;
-                        }) => (
+                        {({ setRef }: { setRef: (instance: HTMLFormElement | null) => void }) => (
                             <form
-                                className={
-                                    is_in_personal_details_modal
-                                        ? 'cfd-personal-details-modal__form'
-                                        : 'cfd-financial-stp-modal__form'
-                                }
+                                className='cfd-personal-details-modal__form'
                                 ref={setRef}
                                 onSubmit={handleSubmit}
                                 autoComplete='off'
+                                noValidate
```

Comment:
Why is this added here? Does it hide the default browser errors that are displayed?

Reply:
yes. to prevent browser validation